### PR TITLE
RC4 changes set and regeneration of spec

### PR DESCRIPTION
**Pull request by SergejMuhic**
Pull request for release.
- regenerated spec repo
- change sets added

### CODE REVIEW

**Review by jmirtsch — APPROVED** [review]
Thanks for the great work Sergej.  Looks fine to me.

**Review by czapplitec — APPROVED** [review]
Great job, @SergejMuhic

File: IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcBuiltElementType/Documentation.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ The _IfcBuiltElementType_ can be instantiated in the case when arbitrary built e
 
 Occurrences of subtypes of the _IfcBuiltElementType_ are represented by instances of the appropriate subtypes of _IfcBuiltElement_.  
 
-> HISTORY&nbsp; HISTORY New entity in IFC2x2.
\ No newline at end of file
+> HISTORY&nbsp; HISTORY New entity in IFC2x2.
```
Comment (pjanck) [inline IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcBuiltElementType/Documentation.md]: Is this intentional, that HISTORY is stated twice?

**Review by pjanck — APPROVED** [review]
I merely scrolled through, did not check for completeness of the changesets.

File: IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcBuiltElementType/Documentation.md (hunk 1 above)
Comment (SergejMuhic) [inline IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcBuiltElementType/Documentation.md:10]:
```suggestion
> HISTORY&nbsp; New entity in IFC2x2.
```

Comment (SergejMuhic) [reply IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcBuiltElementType/Documentation.md]: Thanks. I don't think it is supposed to be like that. I have corrected it.

**Pull request merged**